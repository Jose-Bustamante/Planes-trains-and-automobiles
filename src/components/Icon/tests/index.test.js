import React from "react";
import { render } from "@testing-library/react";
import { ReactComponent as CarIcon } from "../../../assets/car.svg";

import Icon from "../index";

describe("<Icon />", () => {
  it("renders icon with default values", () => {
    const { asFragment } = render(<Icon as={CarIcon} />);
    expect(asFragment()).toMatchSnapshot();
  });
  it("renders icon with custom width", () => {
    const { asFragment } = render(<Icon as={CarIcon} width="16px" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
