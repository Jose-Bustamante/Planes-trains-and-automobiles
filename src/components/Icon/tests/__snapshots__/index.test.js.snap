// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Icon /> renders icon with custom width 1`] = `
<DocumentFragment>
  <svg
    class="sc-bdnylx bmOqsR"
    color="gray"
    fill="gray"
    height="32px"
    stroke-width="0"
    width="16px"
  >
    car.svg
  </svg>
</DocumentFragment>
`;

exports[`<Icon /> renders icon with default values 1`] = `
<DocumentFragment>
  <svg
    class="sc-bdnylx bmhgoD"
    color="gray"
    fill="gray"
    height="32px"
    stroke-width="0"
    width="32px"
  >
    car.svg
  </svg>
</DocumentFragment>
`;
